Palindrome.java

```diff
@@ -0,0 +1,115 @@
+package WEEK1;
```

Comment:
nit: is this week 5?

Reply:
Oops! Sorry for typo.

Reply:
Oops! I forgot to change the package.

Reply:
Thank you @fruby for reviewing. I have made the changes. I used Eclipse for the code.